reference/src/client/src/sa_engine_pkey.c

```diff
@@ -936,6 +936,11 @@ static int pkey_decrypt(
     sa_crypto_cipher_context cipher_context;
     sa_status status = sa_crypto_cipher_init(&cipher_context, cipher_algorithm, SA_CIPHER_MODE_DECRYPT,
             data->private_key, parameters);
+    if (status == SA_STATUS_OPERATION_NOT_SUPPORTED) {
+        ERROR("sa_crypto_cipher_init operation not supported");
+        return -2;
```

Comment:
Why are we not using some defined enum for the return for code readability?

Reply:
Done